webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -1195,7 +1230,7 @@ public static ByteArrayOutputStream requestsToZip(ServletContext context, String
             }
 
             doc.open();
-            addGenericFmcSolutionSheet(docWriter, doc, globalTitle);
+            addGenericFmcSolutionSheet(docWriter, doc, globalTitle, new Locale("en", "US"));
```

Comment:
Why don't we generate a generic fmc solution sheet for every language?

Reply:
Well, we do

https://github.com/campos20/tnoodle/blob/master/webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java#L1378